### DISCUSSION TITLE
Won't Load Settings on Alt

### DISCUSSION THREAD

**Discussion by Entropic8** [SFX-WoW/Masque#186] [Issues] [answered]
Hi,
I have Masque running on my main character and it works fine, but when I try to load exactly the same settings on an alt it will not work. I use ElvUI on both characters with almost the same settings, but for some reason it just will not load or import any settings. Even if I try to set it up from scratch it won't work. Could you look into this?

**Comment by Entropic8** [comment]
To add, OPie support works, but not with ElvUI.

**Comment by StormFX** [comment, marked as answer]
This isn't an issue with Masque. Make sure you've disabled the "AddonSkins" plug-in for ElvUI.
